content/ai/concepts/generative-ai/generative-ai.md

```diff
@@ -0,0 +1,126 @@
+---
+Title: 'Generative-AI'
+Description: Generative AI (GenAI) involves the creation of new and unique digital content. Before GenAI digital content of this level could only be created by human beings. These unique creations are driven by natural language prompts.
+Subjects:
+  - 'Machine Learning'
+  - 'Data Science'
+  - 'Large Language Model'
+  - 'Transformers'
+Tags:
```

Comment:
Only AI is listed on the [Tags list](https://github.com/Codecademy/docs/blob/main/documentation/tags.md).
All the other terms should be added to that file in this PR.